### DISCUSSION TITLE
`option_as_alt` does not really work

### DISCUSSION THREAD

**Discussion by srid** [raphamorim/rio#177] [Q&A] [answered]
I tried both `'both'` and `'left'` as values but it has no effect. <kbd>Option+b</kbd> inserts `∫` and <kbd>Option+f</kbd> inserts `ƒ`.

Is there a way to solve this?

**Comment by raphamorim** [comment]
Hey @srid that option only works on macos. Is the platform that you are using right?

**Reply by srid** [reply]
I'm on a M1 Macbook Pro running macOS 13.4.1 (22F82).

**Reply by raphamorim** [reply]
coolio, I will take a look once return from vacations. Moving this discussion to an issue.

**Reply by raphamorim** [reply, marked as answer]
https://github.com/raphamorim/rio/issues/178

**Reply by srid** [reply]
I just want to say that Rio terminal has been working great, _except_ for one this issue. I'm also looking forward to WASM support, so I can embed this in the browser as part of a https://leptos.dev/ app.
